src/main/java/kai/task/Event.java

```diff
@@ -0,0 +1,30 @@
+package kai.task;
+
+public class Event extends Task {
+    protected String at;
```

Comment:
Rename variable `at` into `startDate` or `beginDate`

Reply:
Definitely! i changed it to a clearer name